docs/key-issues/gpus.md

```diff
@@ -30,54 +30,78 @@ Packaging such projects for PyPI has been, and still is, quite challenging.
+bindings to the APIs (and no bindings for CUDA libraries are provided as of
+yet). Many other projects remain hosted on [NVIDIA's Private PyPI
+Index](https://pypi.org/project/nvidia-pyindex/), which also includes rebuilds
+of TensorFlow and other packages.
 
 A single CUDA version supports a reasonable range of GPU architectures. New
 CUDA versions get released regularly, and - because they come with increased
 performance or new functionality - it may be necessary or desirable to build
 new wheels for that CUDA version. If only the supported CUDA version is
 different between two wheels, the wheel tags and filename will be identical.
 Hence it is not possible to upload more than one of those wheels under the same
-package name. To work around that, a project may either support only one CUDA
-version on PyPI, or create different packages. PyTorch and TensorFlow do the
-former, with TensorFlow supporting only a single CUDA version, and PyTorch
-providing more wheels for other CUDA versions and a CPU-only version in a
-separate wheelhouse (see
+package name.
+
+Historically, this required projects to produce packages specific to CUDA minor
+versions. Projects would either support only one CUDA version on PyPI, or create
+different packages. PyTorch and TensorFlow do the former, with TensorFlow
+supporting only a single CUDA version, and PyTorch providing more wheels for
+other CUDA versions and a CPU-only version in a separate wheelhouse (see
 [pytorch.org/get-started](https://pytorch.org/get-started/locally/)).
 CuPy provides a number of packages: [`cupy`](https://pypi.org/project/cupy/),
 [`cupy-cuda102`](https://pypi.org/project/cupy-cuda102/),
 [`cupy-cuda110`](https://pypi.org/project/cupy-cuda1110/),
 [`cupy-cuda111`](https://pypi.org/project/cupy-cuda111/),
-[`cupy-cuda11x`](https://pypi.org/project/cupy-cuda11x/),
 [`cupy-rocm-4-3`](https://pypi.org/project/cupy-rocm-4-3/),
 [`cupy-rocm-5-0`](https://pypi.org/project/cupy-rocm-5-0/).
-Other projects do similar things - none of it works very well.
-
-GPU packages tend to result in very large wheels. This is true in particular
-for deep learning packages, because they link in
-[cuDNN](https://developer.nvidia.com/cudnn). For example, the most recent
-`manylinux2014` wheels for TensorFlow are 588 MB
-([2.11.0 files](https://pypi.org/project/tensorflow/2.11.0/#files)), and for
-PyTorch those are 890 MB ([1.13.0
-files](https://pypi.org/project/tensorflow/2.11.0/#files)). The problems around
-and causes of GPU wheel sizes were discussed in depth in
-[this Packaging thread on Discourse](https://discuss.python.org/t/what-to-do-about-gpus-and-the-built-distributions-that-support-them/7125).
+This works, but adds maintenance overhead to project developers and consumes
+more storage and network bandwidth for PyPI.org. Moreover, it also prevents
+downstream projects from properly declaring the dependency unless they also
+follow a similar multi-package approach. As of CUDA 11, CUDA promises
```

Comment:
```suggestion
follow a similar multi-package approach. As of CUDA 11.2, CUDA promises
```
If I'm not mistaken, this only started with 11.2

Reply:
Depend on which component you're referring to. For NVRTC, yes.